clang/include/clang/AST/ASTContext.h

```diff
@@ -1261,6 +1262,10 @@ class ASTContext : public RefCountedBase<ASTContext> {
   /// space.
   QualType removeAddrSpaceQualType(QualType T) const;
 
+  /// Return the "other" type-specific discriminator for the given type.
```

Comment:
The comment here looks a bit misleading without knowing what function is doing. Could it be reformulated in some better way?

Reply:
@ojhunt This looks much better, thanks!